validphys2/src/validphys/process_options.py

```diff
@@ -125,8 +125,8 @@ def _dis_xq2map(kin_info):
     """In the old style commondata, the variables in the dataframe were ``x, Q2, y``
     but due to the transformations that happen inside validphys they become ``x, Q, y``
     """
-    x = kin_info["k1"]
-    q = kin_info["k2"]
+    x = kin_info.get_one_of("k1", _Vars.x)
+    q = kin_info.get_one_of("k2", _Vars.Q2)  # this is really Q not Q2
```

Comment:
If it is Q why do you use Q2?

Reply:
The name of the passed variable is Q2 but if one looks at the values they are really Q not Q2. Maybe something is still missing somewhere.

Reply:
You should be using, in the polarized datasets, `identity` so no kinematic override or transformation. When you do that, what goes into validphys is whatever you included in the data file, raw.

It is possible that other DIS datasets (the ones that have k1, k2, k3) use "x" "Q2", "y" but, after going through validphys, the value of k2 gets a `sqrt`.

But for new data we should just use the variables we want. If you need `sqrt(q2)` you can do that explicitly.

Reply:
Could you check that [296b23b](https://github.com/NNPDF/nnpdf/pull/2109/commits/296b23b0523107cdc59776c7e5f5903d552c7f99) is now fine. If so, I will propagate the `kinematics_override` into all the polDIS commondata.

Reply:
Resolved in [e5208f](https://github.com/NNPDF/nnpdf/pull/2109/commits/5e5208feb4bcfec226116f0df5447e73cc9ad528), [9e34271](https://github.com/NNPDF/nnpdf/pull/2109/commits/9e34271580fbddf3115a8e77c1016bd73afc4cd4), [0d680d0](https://github.com/NNPDF/nnpdf/pull/2109/commits/0d680d05e7971b11712bb97df2a21c279bda9217)